### PR TITLE
update covid info

### PR DESCRIPTION
**Pull request by JTeijema**
Adds new info on the covid-19 dataset extension

*edit, didnt include changes to dev in the end, so no merge conflicts should be happening

### CODE REVIEW

**Comment by JTeijema** [comment]
I cant edit #765, but I did pull from that branch. Merge 765 first.

File: docs/source/extensions/extension_covid19.rst [hunk 1]
```diff
@@ -3,74 +3,74 @@
 
 
 ASReview against COVID-19
-=========================
+#########################
```
Comment (J535D165) [inline docs/source/extensions/extension_covid19.rst:6]: Why do you change the header structure?

**Review by J535D165 — CHANGES_REQUESTED** [review]

**Review by Rensvandeschoot — APPROVED** [review]

File: docs/source/extensions/overview_extensions.rst [hunk 2]
```diff
@@ -81,13 +87,19 @@ The following extensions are developed and maintained by the ASReview community:
...
-      then after each 150 iterations, hyperparameter optimisation is conducted 
+      then after each 150 iterations, hyperparameter optimization is conducted 
       to find the most suitable CNN architecture for current iteration.
     - `Github <https://github.com/BartJanBoverhof/asreview-cnn-hpo>`__ 
+    
+* ASReview matrix and vocabulary extractor for TF-IDF and Doc2Vec
+    - An extension for ASReview that adds a tf-idf extractor that saves the
+      matrix and the vocabulary to pickle and JSON respectively, and a doc2vec
+      extractor that grabs the entire doc2vec model. 
+    - `Github <https://github.com/asreview/asreview-extension-vocab-extractor>`__
```
Comment (J535D165) [inline docs/source/extensions/overview_extensions.rst:102]:
```suggestion

```

File: docs/source/extensions/overview_extensions.rst [hunk 3]
```diff
@@ -62,13 +64,17 @@ Dataset extensions
...
+seamlessly integrate your code with the ASReview framework. These extensions can
+extend the software with new models, subcommands, and datasets.
 
 The following extensions are developed and maintained by the ASReview community:
 
 
+Model extensions
+~~~~~~~~~~~~~~~~
+
+
```
Comment (J535D165) [inline docs/source/extensions/overview_extensions.rst:77]:
```suggestion
```

File: docs/source/extensions/extension_covid19.rst [hunk 4]
```diff
@@ -8,69 +8,69 @@ ASReview against COVID-19
...
-the WHO COVID-19 database of publications, the preprint servers bioRxiv and
-medRxiv and papers contributed by specific publishers.
+The datasets are updated in ASReview extension shortly after a release by
+the Allen Institute for AI.
 
-The dataset is available shortly after a release by the Allen Institute for AI.
-
-Pre-print dataset
+Preprint dataset
 -----------------
```
Comment (J535D165) [inline docs/source/extensions/extension_covid19.rst:57]:
```suggestion
----------------
```

**Review by J535D165 — APPROVED** [review]

**Pull request merged**